microsoft-edge/webview2/how-to/debug.md

```diff
@@ -101,148 +110,169 @@ To debug your WebView2 app:
 
    :::image type="content" source="./media/console.png" alt-text="Debug Console in Visual Studio." lightbox="./media/console.png":::
 
-> [!NOTE]
-> If you use the WebView2 [SetVirtualHostNameToFolderMapping](/dotnet/api/microsoft.web.webview2.core.corewebview2.setvirtualhostnametofoldermapping) method, the debugger in Visual Studio 2019 doesn't understand the virtual source path mapping, so breakpoints don't work correctly.  This source path mapping works when you use the debugger in Visual Studio Code.
 
+#### Virtual source path mapping not supported in Visual Studio 2019
+
+If you use the WebView2 [SetVirtualHostNameToFolderMapping](/dotnet/api/microsoft.web.webview2.core.corewebview2.setvirtualhostnametofoldermapping) method, the debugger in Visual Studio 2019 doesn't understand the virtual source path mapping, so breakpoints don't work correctly.
 
+Virtual source path mapping works when you use the debugger in Visual Studio Code.<!-- or in Visual Studio later than 2019. -->
```

Comment:
Does virtual source path mapping work when you use the debugger in Visual Studio later than 2019?